File: 00.ProgramingBasics/05.InputOutputConsole/06.QuadraticEquation/QuadraticEquation.cs
using System;


class QuadraticEquation
{
    static void Main()
    {
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double c = double.Parse(Console.ReadLine());
        double d = b * b - 4 * a * c;
        if (d <0)
        {
            Console.WriteLine("No real roots");
        }
        else if (d == 0)
        {
            Console.WriteLine("x1 = x1 = {0}", (-b / (2 * a)));
        }
        else
        {
            Console.WriteLine("x1 = {0}; x2 = {1}", ((-b + Math.Sqrt(d)) / (2 * a)), ((-b - Math.Sqrt(d)) / (2 * a)));
        } 

    }
}

File: 05.JavaScript/Homeworks/02.JavaScriptSyntax/01.QuadraticEquation/QuadraticEquation.js
var a = prompt("enter value of a");
var b = prompt("enter value of b");
var c = prompt("enter value of c");
var d = b * b - 4 * a * c;

if (d < 0) {
    console.log("No real roots");
} else if (d === 0) {
    console.log("x = " + (-b / 2 * a));
} else {
    console.log("x1 = " + (-b - (Math.sqrt(d))) / (2 * a));
    console.log("x2 = " + (-b + (Math.sqrt(d))) / (2 * a));
}